dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -1394,51 +1393,28 @@ public void testAppToolCollections() throws Exception {
         assertTrue((collection.getEntries().stream().anyMatch(entry -> Objects.equals(entry.getId(), appTool.getId()))));
     }
 
-    /*
-     * TODO: reimplement
-     * This test broke when merged from a 1.12 hotfix into 1.13, because the test pushed a .dockstore.yml that
-     * contains two workflows with the same name, and 1.13 checks for duplicate names in .dockstore.yml, and
-     * generates an error about the duplicate names, instead.  To reimplement, we'll need to load a workflow
-     * into the db, then push a branch that updates it, containing a workflow with the same name but different
-     * descriptor language.
-     */
-    @Ignore
     @Test
     public void testDifferentLanguagesWithSameWorkflowName() throws Exception {
         CommonTestUtilities.cleanStatePrivate2(SUPPORT, false, testingPostgres);
         final io.dockstore.openapi.client.ApiClient webClient = getOpenAPIWebClient(BasicIT.USER_2_USERNAME, testingPostgres);
         io.dockstore.openapi.client.api.WorkflowsApi workflowClient = new io.dockstore.openapi.client.api.WorkflowsApi(webClient);
         io.dockstore.openapi.client.api.UsersApi usersApi = new io.dockstore.openapi.client.api.UsersApi(webClient);
 
+        // Add a WDL version of a workflow should pass.
+        workflowClient.handleGitHubRelease("refs/heads/sameWorkflowName-WDL", installationId, workflowDockstoreYmlRepo, BasicIT.USER_2_USERNAME);
```

Comment:
If I'm understanding the original test description correctly, this should be testing one GitHub release for a .dockstore.yml containing two workflows, one CWL and one WDL, with the same name. The test should verify that one of those workflows is processed even though there is an error. Your test is testing two GitHub releases, each .dockstore.yml containing only one workflow. 

Tagging @svonworl because I think he wrote the original test

Reply:
Yes, that was the original test but the issue was that the 1.13 release now checks if .dockstore.yml contains duplicate named workflows. So to make this test runnable again, I had to make two different .dockstore.yml files and push each workflow separately if that makes sense.